apps/climatemappedafrica/src/payload/blocks/PageHero.js

```diff
@@ -0,0 +1,44 @@
+import richText from "../fields/richText";
+
+const PageHero = {
+  slug: "page-hero",
+  imageURL: "/images/cms/blocks/hero.png",
+  imageAltText: "Used in homepage",
+  labels: {
+    singular: "Site Hero",
+    plural: "Site Hero",
+  },
```

Comment:
Looking at the labels, shouldn't the component be named SiteHero? I'm assuming the implication is the whole site will have just one of these.